### PR TITLE
Bump up SeawaterPolynomials to v0.3.4

### PR DESCRIPTION
**Pull request by glwagner**

### CODE REVIEW

**Comment by glwagner** [comment]
cc @wsmoses @jlk9 lets watch this

File: Project.toml [hunk 1]
```diff
@@ -70,7 +70,7 @@ Pkg = "1.9"
 Printf = "1.9"
 Random = "1.9"
 Rotations = "1.0"
-SeawaterPolynomials = "0.3.2"
+SeawaterPolynomials = "0.3.4"
```
Comment (navidcy) [inline Project.toml:73]: we need to change the version in the Manifest

**Review by navidcy — APPROVED** [review]

**Comment by glwagner** [comment]
Ok good. I'm gonna merge anyways.

**Comment by navidcy** [comment]
> Ok good. I'm gonna merge anyways.

I did it anyways.

**Pull request merged**